File: CTDK_Python/066/_066.py
n = int(input("Nhap n: "))

flag = 0;
t = n
while(t != 0):
    dv = t % 10
    if dv % 2 == 0:
        flag = 1
    t = t // 10

if flag == 1:   
    print("TT chan")
else:
    print("Khong TT chan")

File: CTDK/066/066.cpp
#include <iostream>
using namespace std;

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	int flag = 0;
	int t = n;
	while (t != 0)
	{
		int dv = t % 10;
		if (dv % 2 == 0)
			flag = 1;
		t = t / 10;
	}

	if (flag == 1)
		cout << "Ton tai chu so chan" << endl;
	else
		cout << "Khong ton tai chu so chan" << endl;

	return 0;
}